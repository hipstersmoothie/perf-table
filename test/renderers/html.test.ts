import renderHTML from '../../src/renderers/html';

const data = [
  ['name', 'age', 'pizza topping'],
  ['burt', '40', 'anchovies'],
  ['gert', '35', 'pineapple'],
  ['john', '18', 'basil']
];

test('renderHTML', () => {
  expect(renderHTML(data)).toMatchSnapshot();
});
