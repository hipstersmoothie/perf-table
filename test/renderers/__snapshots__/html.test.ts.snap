// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderHTML 1`] = `
"
  <table>
    <thead>
      <tr>
        <th>name</th>
<th>age</th>
<th>pizza topping</th>
      </tr>
    </thead>
    <tbody>
      
          <tr>
            <td>burt</td>
<td>40</td>
<td>anchovies</td>
          </tr>
        

          <tr>
            <td>gert</td>
<td>35</td>
<td>pineapple</td>
          </tr>
        

          <tr>
            <td>john</td>
<td>18</td>
<td>basil</td>
          </tr>
        
    </tbody>
  </table>
"
`;
